### ISSUE TITLE
Sieve error after latest upgrade

### ISSUE THREAD

**Issue by aukfood** [mailcow/mailcow-dockerized#5664]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/mailcow/mailcow-dockerized/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... I understand that not following the below instructions will result in immediate closure and/or deletion of my issue.
- [X] ... I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [X] ... I have understood that answers are voluntary and community-driven, and not commercial support.
- [X] ... I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

### Description

```plain text
Hello, I upgraded mailcow on latest version and so now I have problems with dovecot delivry with sieve errors.
```

### Logs:

```plain text
info	lmtp(mail@domaine.com)<39188><yLGTH3OdrmUUmQAAaXmRoA>: sieve: Execution of script /var/vmail/sieve/mail@domaine.com.sieve failed, but implicit keep was successful (user logfile /var/vmail/sieve/mail@domaine.com.sieve.log may reveal additional details)

/var/vmail/sieve/mail@domaine.com.sieve.log
error: msgid=<20240122165832.632E4290081D@domaine.com>: redirect action: failed to redirect message to <user@doomaine.com>: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure).
```

### Steps to reproduce:

```plain text
1. mail arrive to postfix
2. mail arrive to dovecot
3. dovecot can't deliver mail
```

### Which branch are you using?

master

### Operating System:

debian 11

### Server/VM specifications:

15Go mem, 8cpus

### Is Apparmor, SELinux or similar active?

no

### Virtualization technology:

kvm with proxmox

### Docker version:

23.0.0

### docker-compose version or docker compose version:

docker-compose-plugin                    2.15.1-1~debian.10~buster

### mailcow version:

2024-01b

### Reverse proxy:

non

### Logs of git diff:

```plain text
git diff origin/master 
diff --git a/data/assets/ejabberd/sqlite/sqlite.db b/data/assets/ejabberd/sqlite/sqlite.db
new file mode 100644
index 00000000..981ba6b9
Binary files /dev/null and b/data/assets/ejabberd/sqlite/sqlite.db differ
diff --git a/data/conf/clamav/freshclam.conf b/data/conf/clamav/freshclam.conf
index cfb497e9..c66e758d 100644
--- a/data/conf/clamav/freshclam.conf
+++ b/data/conf/clamav/freshclam.conf
@@ -16,4 +16,11 @@ ConnectTimeout 20
 ReceiveTimeout 20
 TestDatabases yes
 Bytecode yes
-
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfo.hdb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfo.ign2
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/javascript.ndb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/spam_marketing.ndb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfohtml.hdb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfoascii.hdb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfoandroid.hdb
+DatabaseCustomURL http://www.securiteinfo.com/get/signatures/5fd3b48287b1a5dcc24f9e84825b3dfa51ad70a0e1ca8b968106b3443e2eb295b8e56018e5c244a3f6eb1fa411f4300bcd41a27934fe0ae64792066992014020/securiteinfoold.hdb
diff --git a/data/conf/ejabberd/autogen/ejabberd_acl.yml b/data/conf/ejabberd/autogen/ejabberd_acl.yml
new file mode 100644
index 00000000..21db66a4
--- /dev/null
+++ b/data/conf/ejabberd/autogen/ejabberd_acl.yml
@@ -0,0 +1 @@
...
```

### Logs of iptables -L -vn:

```plain text
# Warning: iptables-legacy tables present, use iptables-legacy to see them
Chain INPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
 283K  200M MAILCOW    all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* mailcow */
  589 25440 DROP       all  --  *      *       0.0.0.0/0            0.0.0.0/0            match-set crowdsec-blacklists src

Chain FORWARD (policy DROP 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
3135K 1802M MAILCOW    all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* mailcow */
3144K 2010M DOCKER-USER  all  --  *      *       0.0.0.0/0            0.0.0.0/0           
3144K 2010M DOCKER-ISOLATION-STAGE-1  all  --  *      *       0.0.0.0/0            0.0.0.0/0           
1927K 1496M ACCEPT     all  --  *      br-mailcow  0.0.0.0/0            0.0.0.0/0            ctstate RELATED,ESTABLISHED
 122K 9312K DOCKER     all  --  *      br-mailcow  0.0.0.0/0            0.0.0.0/0           
1094K  505M ACCEPT     all  --  br-mailcow !br-mailcow  0.0.0.0/0            0.0.0.0/0           
 114K 8809K ACCEPT     all  --  br-mailcow br-mailcow  0.0.0.0/0            0.0.0.0/0           
    0     0 ACCEPT     all  --  *      docker0  0.0.0.0/0            0.0.0.0/0            ctstate RELATED,ESTABLISHED
    0     0 DOCKER     all  --  *      docker0  0.0.0.0/0            0.0.0.0/0           
    0     0 ACCEPT     all  --  docker0 !docker0  0.0.0.0/0            0.0.0.0/0           
    0     0 ACCEPT     all  --  docker0 docker0  0.0.0.0/0            0.0.0.0/0           

Chain OUTPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain DOCKER (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.249         tcp dpt:6379
    0     0 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.5           tcp dpt:8983
    0     0 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.6           tcp dpt:3306
    0     0 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:12345
    0     0 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:4190
   32  1872 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:995
 2905  173K ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:993
  804 44943 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.8           tcp dpt:443
  283 15662 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.8           tcp dpt:80
  289 17683 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:143
   28  1576 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.250         tcp dpt:110
  290 17200 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.253         tcp dpt:587
 2686  160K ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.253         tcp dpt:465
 1229 70704 ACCEPT     tcp  --  !br-mailcow br-mailcow  0.0.0.0/0            172.22.1.253         tcp dpt:25

Chain DOCKER-ISOLATION-STAGE-1 (1 references)
 pkts bytes target     prot opt in     out     source               destination         
1094K  505M DOCKER-ISOLATION-STAGE-2  all  --  br-mailcow !br-mailcow  0.0.0.0/0            0.0.0.0/0           
    0     0 DOCKER-ISOLATION-STAGE-2  all  --  docker0 !docker0  0.0.0.0/0            0.0.0.0/0           
1412M  817G RETURN     all  --  *      *       0.0.0.0/0            0.0.0.0/0           

Chain DOCKER-ISOLATION-STAGE-2 (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DROP       all  --  *      br-mailcow  0.0.0.0/0            0.0.0.0/0           
    0     0 DROP       all  --  *      docker0  0.0.0.0/0            0.0.0.0/0           
 412M  275G RETURN     all  --  *      *       0.0.0.0/0            0.0.0.0/0           

Chain DOCKER-USER (1 references)
 pkts bytes target     prot opt in     out     source               destination         
1412M  817G RETURN     all  --  *      *       0.0.0.0/0            0.0.0.0/0           

Chain MAILCOW (2 references)
 pkts bytes target     prot opt in     out     source               destination         
  311 18580 DROP       all  --  *      *       141.98.11.0/24       0.0.0.0/0           
    0     0 DROP       all  --  *      *       45.134.23.234        0.0.0.0/0           
    0     0 DROP       all  --  *      *       100.120.12.169       0.0.0.0/0           
    0     0 DROP       all  --  *      *       77.247.110.102       0.0.0.0/0           
    0     0 DROP       all  --  *      *       40.115.207.229       0.0.0.0/0           
    0     0 DROP       all  --  *      *       212.192.219.124      0.0.0.0/0           
    0     0 DROP       all  --  *      *       185.239.242.43       0.0.0.0/0           
    0     0 DROP       all  --  *      *       193.56.29.211        0.0.0.0/0           
    0     0 DROP       all  --  *      *       100.120.22.107       0.0.0.0/0
```

### Logs of ip6tables -L -vn:

```plain text
# Warning: ip6tables-legacy tables present, use ip6tables-legacy to see them
Chain INPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
 1724  115K MAILCOW    all      *      *       ::/0                 ::/0                 /* mailcow */
    0     0 DROP       all      *      *       ::/0                 ::/0                 match-set crowdsec6-blacklists src

Chain FORWARD (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
 186K  252M MAILCOW    all      *      *       ::/0                 ::/0                 /* mailcow */
 187K  252M DOCKER-USER  all      *      *       ::/0                 ::/0                
 140M  164G DOCKER-ISOLATION-STAGE-1  all      *      *       ::/0                 ::/0                
 140M  164G DOCKER     all      *      br-mailcow  ::/0                 ::/0                
 128M  164G ACCEPT     all      *      br-mailcow  ::/0                 ::/0                 ctstate RELATED,ESTABLISHED
    0     0 ACCEPT     all      br-mailcow !br-mailcow  ::/0                 ::/0                
  12M  889M ACCEPT     all      br-mailcow br-mailcow  ::/0                 ::/0                

Chain OUTPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain DOCKER (1 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::b  tcp dpt:995
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::b  tcp dpt:110
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::b  tcp dpt:143
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::b  tcp dpt:4190
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::b  tcp dpt:993
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::c  tcp dpt:443
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::c  tcp dpt:80
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::f  tcp dpt:587
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::f  tcp dpt:25
    0     0 ACCEPT     tcp      !br-mailcow br-mailcow  ::/0                 fd4d:6169:6c63:6f77::f  tcp dpt:465

Chain DOCKER-ISOLATION-STAGE-1 (1 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DOCKER-ISOLATION-STAGE-2  all      br-mailcow !br-mailcow  ::/0                 ::/0                
 187K  252M RETURN     all      *      *       ::/0                 ::/0                

Chain DOCKER-ISOLATION-STAGE-2 (1 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DROP       all      *      br-mailcow  ::/0                 ::/0                
    0     0 RETURN     all      *      *       ::/0                 ::/0                

Chain DOCKER-USER (1 references)
 pkts bytes target     prot opt in     out     source               destination         
 140M  164G RETURN     all      *      *       ::/0                 ::/0                

Chain MAILCOW (2 references)
 pkts bytes target     prot opt in     out     source               destination
```

### Logs of iptables -L -vn -t nat:

```plain text
# Warning: iptables-legacy tables present, use iptables-legacy to see them
Chain PREROUTING (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
7383K  426M DOCKER     all  --  *      *       0.0.0.0/0            0.0.0.0/0            ADDRTYPE match dst-type LOCAL

Chain INPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain OUTPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DOCKER     all  --  *      *       0.0.0.0/0           !127.0.0.0/8          ADDRTYPE match dst-type LOCAL

Chain POSTROUTING (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
 189K   15M MASQUERADE  all  --  *      !br-mailcow  172.22.1.0/24        0.0.0.0/0           
    0     0 MASQUERADE  all  --  *      !docker0  172.17.0.0/16        0.0.0.0/0           
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.249         172.22.1.249         tcp dpt:6379
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.5           172.22.1.5           tcp dpt:8983
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.6           172.22.1.6           tcp dpt:3306
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:12345
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:4190
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:995
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:993
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.8           172.22.1.8           tcp dpt:443
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.8           172.22.1.8           tcp dpt:80
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:143
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.250         172.22.1.250         tcp dpt:110
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.253         172.22.1.253         tcp dpt:587
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.253         172.22.1.253         tcp dpt:465
    0     0 MASQUERADE  tcp  --  *      *       172.22.1.253         172.22.1.253         tcp dpt:25

Chain DOCKER (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 RETURN     all  --  br-mailcow *       0.0.0.0/0            0.0.0.0/0           
    0     0 RETURN     all  --  docker0 *       0.0.0.0/0            0.0.0.0/0           
    0     0 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            127.0.0.1            tcp dpt:7654 to:172.22.1.249:6379
    0     0 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            127.0.0.1            tcp dpt:18983 to:172.22.1.5:8983
    0     0 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            127.0.0.1            tcp dpt:13306 to:172.22.1.6:3306
    0     0 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            127.0.0.1            tcp dpt:19991 to:172.22.1.250:12345
    0     0 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:4190 to:172.22.1.250:4190
   32  1872 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:995 to:172.22.1.250:995
 2923  174K DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:993 to:172.22.1.250:993
  811 45363 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:443 to:172.22.1.8:443
  284 15722 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:80 to:172.22.1.8:80
  291 17811 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:143 to:172.22.1.250:143
   30  1696 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:110 to:172.22.1.250:110
  290 17200 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:587 to:172.22.1.253:587
 4576  274K DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:465 to:172.22.1.253:465
 1238 71244 DNAT       tcp  --  !br-mailcow *       0.0.0.0/0            0.0.0.0/0            tcp dpt:25 to:172.22.1.253:25
```

### Logs of ip6tables -L -vn -t nat:

```plain text
# Warning: ip6tables-legacy tables present, use ip6tables-legacy to see them
Chain PREROUTING (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DOCKER     all      *      *       ::/0                 ::/0                 ADDRTYPE match dst-type LOCAL

Chain INPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain OUTPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DOCKER     all      *      *       ::/0                !::1                  ADDRTYPE match dst-type LOCAL

Chain POSTROUTING (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 MASQUERADE  all      *      br-mailcow  ::/0                 ::/0                 ADDRTYPE match dst-type LOCAL
    0     0 MASQUERADE  all      *      !br-mailcow  fd4d:6169:6c63:6f77::/64  ::/0                
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:110
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:143
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:110
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:143
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:4190
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:993
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:995
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::f  fd4d:6169:6c63:6f77::f  tcp dpt:25
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::f  fd4d:6169:6c63:6f77::f  tcp dpt:465
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::f  fd4d:6169:6c63:6f77::f  tcp dpt:587
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::10  fd4d:6169:6c63:6f77::10  tcp dpt:25
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::10  fd4d:6169:6c63:6f77::10  tcp dpt:465
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::10  fd4d:6169:6c63:6f77::10  tcp dpt:587
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:80
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:443
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:4190
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:993
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::b  fd4d:6169:6c63:6f77::b  tcp dpt:995
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:80
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::c  fd4d:6169:6c63:6f77::c  tcp dpt:443
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::11  fd4d:6169:6c63:6f77::11  tcp dpt:25
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::11  fd4d:6169:6c63:6f77::11  tcp dpt:465
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::11  fd4d:6169:6c63:6f77::11  tcp dpt:587
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::f  fd4d:6169:6c63:6f77::f  tcp dpt:443
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::f  fd4d:6169:6c63:6f77::f  tcp dpt:80
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::e  fd4d:6169:6c63:6f77::e  tcp dpt:143
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::e  fd4d:6169:6c63:6f77::e  tcp dpt:4190
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::e  fd4d:6169:6c63:6f77::e  tcp dpt:993
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::e  fd4d:6169:6c63:6f77::e  tcp dpt:995
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::e  fd4d:6169:6c63:6f77::e  tcp dpt:110
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::d  fd4d:6169:6c63:6f77::d  tcp dpt:25
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::d  fd4d:6169:6c63:6f77::d  tcp dpt:465
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::d  fd4d:6169:6c63:6f77::d  tcp dpt:587
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::d  fd4d:6169:6c63:6f77::d  tcp dpt:443
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::d  fd4d:6169:6c63:6f77::d  tcp dpt:80
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::12  fd4d:6169:6c63:6f77::12  tcp dpt:110
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::12  fd4d:6169:6c63:6f77::12  tcp dpt:143
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::12  fd4d:6169:6c63:6f77::12  tcp dpt:4190
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::12  fd4d:6169:6c63:6f77::12  tcp dpt:993
    0     0 MASQUERADE  tcp      *      *       fd4d:6169:6c63:6f77::12  fd4d:6169:6c63:6f77::12  tcp dpt:995

Chain DOCKER (2 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 RETURN     all      br-mailcow *       ::/0                 ::/0                
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:995 to:[fd4d:6169:6c63:6f77::b]:995
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:110 to:[fd4d:6169:6c63:6f77::b]:110
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:143 to:[fd4d:6169:6c63:6f77::b]:143
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:4190 to:[fd4d:6169:6c63:6f77::b]:4190
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:993 to:[fd4d:6169:6c63:6f77::b]:993
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:443 to:[fd4d:6169:6c63:6f77::c]:443
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:80 to:[fd4d:6169:6c63:6f77::c]:80
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:587 to:[fd4d:6169:6c63:6f77::f]:587
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:25 to:[fd4d:6169:6c63:6f77::f]:25
    0     0 DNAT       tcp      !br-mailcow *       ::/0                 ::/0                 tcp dpt:465 to:[fd4d:6169:6c63:6f77::f]:465
```

### DNS check:

```plain text
172.64.155.249
104.18.32.7
```

**Comment by DerLinkman** [comment]
Where do you setup this Sieve Filter?

In mailcow UI or SOGo?

**Comment by aukfood** [comment]
@DerLinkman in Roundcube. It is better to create them in mailcow ui ?

**Comment by DerLinkman** [comment]
On my instance the filters in mailcow UI and SOGo works fine. However Roundcube is a Community driven Module and is not officially supported by us. So if your error only exists with roundcube it maybe be needed to rewrite the Docs about Roundcube accordingly.

Try to set them up in the mailcow UI and see if they work there or not.

However your Logfile say that it cannot contact your Postfix Server. Is your Postfix healthy and running in general?

**Comment by aukfood** [comment]
@DerLinkman yes my postfix works fine, i'll go this day to see sieve in mailcow and remove them in roundcube

**Comment by kwisatz** [comment]
I'm seeing the same issue since the last update. I have about 30 mails in the queue, all stating:

> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))

I am indeed also running a roundcube container next to the SoGo container, however, I think this is either related to #5668 (see dovecot logs) or my server having been blocked by outlook.

#### Postfix logs 

```
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[706]: 890EB3023EB: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=130376, delays=130376/0.02/0.02/0.01, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[704]: 6522430A835: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=87525, delays=87525/0.02/0.01/0.05, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[705]: 68006301CB9: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=177299, delays=177299/0.02/0.01/0.12, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[707]: 6AC73302163: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=133935, delays=133935/0.03/0.02/0.12, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <mbressler@zev.lu> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[708]: CAB37301AC7: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=20388, delays=20388/0.02/0.02/0.12, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[710]: 274C930A897: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=51042, delays=51042/0.03/0.02/0.11, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
postfix-mailcow-1  | Jan 23 22:11:07 050ddf1649f8 postfix/lmtp[709]: 86EBA300ECA: to=<email@domain.tld>, relay=dovecot[172.22.1.250]:24, delay=169928, delays=169928/0.03/0.02/0.11, dsn=4.2.0, status=deferred (host dovecot[172.22.1.250] said: 451 4.2.0 <email@domain.tld> Execution of Sieve filters was aborted due to temporary failure (in reply to end of DATA command))
```

#### Dovecot logs

(anotherhost.tld is also an outlook.com host)

```
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3116><Y72AHpcssGUsDAAA4SvNQA>: Error: smtp-client: conn postfix:588 ([fd4d:6169:6c63:6f77::13]:588) [1]: connect(postfix:588) failed: Connection refused
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3116><Y72AHpcssGUsDAAA4SvNQA>: Error: sieve: msgid=<INX.126d21bb97c6e10d730ce76767d.40d5e.6953b.edf7.18d3041f932bd6@news.beltz.de>: redirect action: failed to redirect message to <user@anotherhost.tld: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3116><Y72AHpcssGUsDAAA4SvNQA>: Error: sieve: Execution of script /var/vmail/sieve/email@domain.tld.sieve was aborted due to temporary failure (user logfile /var/vmail/sieve/email@domain.tld.sieve.log may reveal additional details)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(3118): Connect from 172.22.1.253
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3117><VTcZH5cssGUtDAAA4SvNQA>: Error: smtp-client: conn postfix:588 ([fd4d:6169:6c63:6f77::13]:588) [1]: connect(postfix:588) failed: Connection refused
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3117><VTcZH5cssGUtDAAA4SvNQA>: Error: sieve: msgid=<20240123201025.7b23ae0da3bb26e5@discover.pinterest.com>: redirect action: failed to redirect message to <user@anotherhost.tld: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3117><VTcZH5cssGUtDAAA4SvNQA>: Error: sieve: Execution of script /var/vmail/sieve/email@domain.tld.sieve was aborted due to temporary failure (user logfile /var/vmail/sieve/email@domain.tld.sieve.log may reveal additional details)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(3121): Connect from 172.22.1.253
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<2765><iCpdHZcssGXNCgAA4SvNQA>: Error: smtp-client: conn postfix:588 ([fd4d:6169:6c63:6f77::13]:588) [1]: connect(postfix:588) failed: Connection refused
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<2765><iCpdHZcssGXNCgAA4SvNQA>: Error: sieve: msgid=<CAF2GhT2RBsrs6MDJGJ8rTGBYgZ-v77Vuk8OS-Zovbbcc4356uw@mail.gmail.com>: redirect action: failed to redirect message to <user@hotmail.com>: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<2765><iCpdHZcssGXNCgAA4SvNQA>: Error: sieve: Execution of script /var/vmail/sieve/global_sieve_after.sieve was aborted due to temporary failure
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(2765): Disconnect from 172.22.1.253: Logged out (state=READY)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3118><cq6CH5cssGUuDAAA4SvNQA>: Error: smtp-client: conn postfix:588 ([fd4d:6169:6c63:6f77::13]:588) [1]: connect(postfix:588) failed: Connection refused
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3118><cq6CH5cssGUuDAAA4SvNQA>: Error: sieve: msgid=<QGuC0mTVQXSGbJBZ7QyBpA@geopod-ismtpd-9>: redirect action: failed to redirect message to <user@hotmail.com>: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3118><cq6CH5cssGUuDAAA4SvNQA>: Error: sieve: Execution of script /var/vmail/sieve/global_sieve_after.sieve was aborted due to temporary failure
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3121><zudCIJcssGUxDAAA4SvNQA>: Error: smtp-client: conn postfix:588 ([fd4d:6169:6c63:6f77::13]:588) [1]: connect(postfix:588) failed: Connection refused
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3121><zudCIJcssGUxDAAA4SvNQA>: Error: sieve: msgid=<349717579.4725.1705881870581@mpzhllaep01.internal.six-group.net>: redirect action: failed to redirect message to <user@hotmail.com>: smtp(postfix:588): RCPT TO failed: Failed to connect to remote server (temporary failure)
dovecot-mailcow-1  | Jan 23 21:16:07 bdf8a64056b5 dovecot: lmtp(email@domain.tld)<3121><zudCIJcssGUxDAAA4SvNQA>: Error: sieve: Execution of script /var/vmail/sieve/global_sieve_after.sieve was aborted due to temporary failure

```

**Comment by DerLinkman** [comment]
Yep can confirm it. Thanks to @kwisatz and @dragoangel who analysed it deeper and bring me to this issue. We have to see how we can compensate this. In the worst case every user of mailcow starting with 25.0.0 needs to make a seperate change to their compose.yml. But i have hopes in docker that they will reimplement at least the usage of the enable_ipv6 parameter, otherwise it does not make sense it still exists tbh.

**Comment by kwisatz** [comment]
After removing the `IPv6` subnet configuration as mentioned in #5668 I can confirm that my mailq is back to empty and `dovecot/sieve` is again able to connect to `postfix`.

**Comment by aukfood** [comment]
@kwisatz if i remove ipv6 subnet I have an error with docker-compose. What is the better solution ?

**Comment by DerLinkman** [comment]
> @kwisatz if i remove ipv6 subnet I have an error with docker-compose. What is the better solution ?

What issue do you have?

Logs please.

**Comment by aukfood** [comment]
@DerLinkman a message like ipv6 subnet does not be configured, but I commented all lines with IPV6_SUBNET in docker-compose.

**Comment by aukfood** [comment]
@DerLinkman sorry I forget to disable "enable_ipv6: false" all mail bloqued by sieve are delivred

**Comment by milkmaker** [comment]
This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.
